l10n/en/firefox/whatsnew/whatsnew.ftl

```diff
@@ -23,3 +23,5 @@ whatsnew-release-notes-v2 = Read the <a { $url }>Release Notes</a> to know more
 whatsnew-release-notes = Read the <a href="{ $url }">Release Notes</a> to know more about what’s new in your { -brand-name-firefox } browser.
 
 whatsnew-signoff = <strong>Powered by { -brand-name-mozilla }.</strong> Putting people before profits since 1998.
+
+whatsnew-donate = Donate
```

Comment:
is this needed when the CTA string translations are hardcoded in the several locales directly in the templates?

Reply:
Currently we're going to be opting to use hard-coded conditionals for the translation, but we're keeping this for translators in Smartling to translate when they can